Dockerfile

```diff
@@ -9,10 +17,12 @@ WORKDIR /code
 RUN make deps \
     && BIN=/bin/hermes make build
 
-FROM alpine:3.15
+FROM alpine:3.18
 RUN mkdir /app
 COPY --from=builder /bin/hermes /app
 WORKDIR /app
 
 ENTRYPOINT ./hermes
 EXPOSE 80
+
+USER $USER
```

Comment:
Is this argument persisted from the previous build stage? Would be great if you can verify if the user is runner in the finally built image.

Reply:
Fixed in [0d9d2f0](https://github.com/DeepSourceCorp/hermes/pull/66/commits/0d9d2f03bbd30e0ceb1ab4617f826dc63945ff4d)